service/src/main/kotlin/no/nav/su/se/bakover/service/statistikk/mappers/behandling/BehandlingStatus.kt

```diff
@@ -0,0 +1,20 @@
+package no.nav.su.se.bakover.service.statistikk.mappers.behandling
+
+/**
+ * Statistikk skiller på status og resultat, hvor vi ofte har slått de sammen i en stor enum/sealed.
+ */
+enum class BehandlingStatus(val beskrivelse: String) {
+    SØKNAD_MOTTATT("Søknaden er mottatt og registrert i systemet. En asynkron jobb har blitt startet for å journalføre og lage Gosys-oppgave."),
+    SØKNAD_JOURNALFØRT("Søknaden er journalført. Det er et krav før vi oppretter en saksbehandleroppgave i Gosys."),
+    SØKNAD_OPPRETTET_OPPGAVE("Opprettet oppgave i Gosys.. Det er et krav før en saksbehandler kan starte saksbehandlingen."),
```

Comment:
Er dette relevant for statistikk-gjengen? Føles ikke som det kommuniserer noe som angår "tilstanden" på en behandling?

Reply:
Fjernet.